fake_data_generation/README.md

```diff
@@ -0,0 +1,97 @@
+`hourly_elective_prob.json`| More information about this file can be found [here.](../src/data_preparation/README.md)
+`specialty_info.json`| More information about this file can be found [here.](../src/data_preparation/README.md)
+
+The fake data is intended for us with the following applications:
+- As an example of how data needs to be formatted to be passed into the models.
+- To ensure the files are being generated correctly to test GUI setup.
+- To test the setup and running of the repo.
+
+The following should be noted with regards to using the fake data generators and the artefacts produced by using fake data:
+- *DO NOT* use the model artefacts (`.pkl` files) generated from the fake data to make predictions which will be used in any real world application.
+- *DO NOT* use the fake data generated to inform any insights to be applied to a real world setting.
+- *DO NOT* use the fake data to test the performance of the model.
+
+The data is generated completely randomly, with each field having random values generated independently of other fields. This generator was created having never been exposed to the real data.
+
+Note a number of the fields will show only test categories and may not be reflective of the types of categories or number of categories types in each field. However the formatting of the fields is reflective as an example of what is required to run the models.
+
+
+## `generate_fake_data.py`
+
+The categories for the fields in `generate_fake_data.py` can be found in [patient_fake_data_categories.json](config/fake_data_categories/patient_fake_data_categories.json). The categories specified in this file are randomly chosen to fill the corresponding fields in the fake data. This is done in `generate_fake_data.py`. 
+
+Data fields that do not have their categories specified in [patient_fake_data_categories.json](../config/fake_data_categories/patient_fake_data_categories.json) or are not categorical variables have the fake data required generated in `generate_fake_data.py` line by line to show how the data is being generated for each field. In `generate_fake_data.py` the fields are split into the field data type (e.g. str, int) required for the models to train.
+
+The fake data generator creates a new directory in the directory the script is ran, the four files described above are then stored in the new directory. An example layout after `generate_fake_data.py` has run with the default new directory name (`fake_data_files`) can be seen below:
+```
+Folder
+│   generate_fake_data.py
+|
+└───fake_data_files
+    │   patient_df.csv
+    │   historic_admissions.csv
+    |   hourly_elective_prob.json
+    |   specialty_info.json
+```
+
+
+### How to run
+Before running ensure your environment is set up as described in: [README.md](../README.md).
```

Comment:
Can you clarify this needs to be in "development" mode (https://github.com/nhsx/skunkworks-bed-allocation/blob/2fd925ff14fe207089f9838a86ea501c029b9df0/README.md?plain=1#L63)